addons/account_asset/wizard/account_asset_change_duration.py

```diff
@@ -56,7 +56,7 @@ def default_get(self, fields):
         if 'method_end' in fields and asset.method_time == 'end':
             res.update({'method_end': asset.method_end})
         if self.env.context.get('active_id'):
-            res['asset_method_time'] = self._get_asset_method_time()
+            res['asset_method_time'] = self._compute_asset_method_time()
```

Comment:
We set None as a value in the dict ? What is its use ? Shouldn't it be res[..] = self.asset_method_time ?